src/offset/utc.rs

```diff
@@ -102,6 +107,10 @@ impl Offset for Utc {
     fn fix(&self) -> FixedOffset {
         FixedOffset::east(0)
     }
+    #[cfg(feature = "std")]
```

Comment:
I don't think adding this makes sense.

Reply:
If FixedOffset had it's own now that didn't need Utc but required systemtime which is in std...